source/config/configFlags.py

```diff
@@ -225,3 +225,49 @@ def _displayStringLabels(self):
 			# Translators: This is a label for the automatic update behaviour for add-ons.
 			self.DISABLED: _("Disabled"),
 		}
+
+
+@unique
+class OutputMode(DisplayStringIntEnum):
+	"""Enumeration for ways to output information, such as formatting.
+	Use OutputMode.MEMBER.value to compare with the config;
+	use OutputMode.MEMBER.displayString in the UI for a translatable description of this member.
+
+	This is currently implemented as an IntEnum due to limitations iterating over IntFlag members. However, an attempt should be made to adhere to the convention that single output modes are powers of 2, and combinations of output modes are bitwise ORs of these values.
+	"""
+
+	OFF = 0b0
+	SPEECH = 0b01
+	BRAILLE = 0b10
+	SPEECH_AND_BRAILLE = SPEECH | BRAILLE
+
+	@property
+	def _displayStringLabels(self):
+		return {
+			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
+			self.OFF: _("Off"),
+			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
+			self.SPEECH: _("Speech"),
+			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
+			self.BRAILLE: _("Braille"),
+			# Translators: A label for an option to choose a method of reporting information, e.g. font attributes.
+			self.SPEECH_AND_BRAILLE: _("Speech and braille"),
+		}
+
+	@enum_property
+	def inSpeech(self) -> bool:
+		"""Check if the output mode includes speech.
+
+		Returns:
+				bool: True if the output mode includes speech, False otherwise.
+		"""
+		return self & self.SPEECH != 0
+
+	@enum_property
+	def inBraille(self) -> bool:
+		"""Check if the output mode includes braille.
+
+		Returns:
+				bool: True if the output mode includes braille, False otherwise.
+		"""
+		return self & self.BRAILLE != 0
```

Comment:
I think this level of abstraction is unnecessary and more confusing. I don't think the meaning of `inSpeech` is particularly clear without reading the docstring. I think performing `& self.SPEECH` directly is fine and much more clear and simple.